<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepositionFramesDialog</class>
 <widget class="QDialog" name="RepositionFramesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reposition Frames</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labRepositionFrames">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listSelectedFrames"/>
   </item>
   <item>
    <widget class="QLabel" name="labInfo">
     <property name="text">
      <string>(Please move selection to desired destination.)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labRepositioned">
     <property name="text">
      <string>Reposition (x,y): </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbOtherLayers">
     <property name="text">
      <string>Reposition on other layers?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbSameKeyframes">
     <property name="text">
      <string>Same keyframes as selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbAllKeyframes">
     <property name="text">
      <string>All keyframes on layer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listSelectedLayers">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReposition">
       <property name="text">
        <string>Reposition</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
